inc/invitation.class.php

```diff
@@ -47,6 +47,10 @@ class PluginFlyvemdmInvitation extends CommonDBTM {
     */
    protected $user;
 
+   /**
+    * Gets the status of the invitations
```

Comment:
The method returns the possible values for the status of an invitation. It can be used to upgrade the database if someday a new status is added, or for a sanitization need.

So this method does not returns the status of an invitation. It returns the possbile statuses that an invitation can have.

This applies to other similar methods in the plugin. (I retrieved this from Formcreator plugin; initially designed by @orthagh, and I like it)